Crowdfunding fixes

ecino opened the pull request.

ecino committed:
CO-3426 Add project name and description in frontend update form

- Notify responsible if those values are changed.
- Add mail thread and mail activities to projects and participants.

ecino committed:
Remove presentation_video from participant and personal_motivation from participant

- Those duplicate fields don't make sense on both project and participant.
- Fix project page to display correctly the information whether it's a personal or collective project.

ecino committed:
CO-3500 Corrections for crowdfunding projects and participant pages

- Share links for participant
- Website links and published states are more standard
- Link to go to project page from participant page

ecino committed:
CO-3502 Allow participant to modify their profile pictures

ecino committed:
Update translations

Theyiot commented on crowdfunding_compassion/forms/project_update_form.py:
```diff
@@ -74,4 +78,27 @@ def form_before_create_or_update(self, values, extra_values):
+                    note=f"{self.main_object.project_owner_id.name} updated the "
+                    f"name and description of the project. "
+                    f"Please check if the information is good enough.",
+                    user_id=user.id
```
Are we sure that there will be a user in all cases (since we get the user using `:1` instead of simply `0`) ? I don't know if it would be overkill to put the `activity_schedule()` inside a `if user:`.

ecino replied:
If notify_ids is set there should be no reason that there is no user so I think we can keep it like that. Thanks for checking.

Theyiot reviewed (approved):
Really good modifications and fixes. You did catch quite a few subtle issues !

I simply added a comment on a little concern I have, otherwise it looks good to me. I did not check the translations, though.

ecino committed:
Improve together my_account view

ecino committed:
Small corrections discussed with Olivier and Merve

- Link back to project page instead of project description on the participant page
- Link to project more obvious in donation page
